tests/vmi_monitoring_test.go

```diff
@@ -78,3 +75,16 @@ var _ = Describe("[sig-compute]Health Monitoring", func() {
 		})
 	})
 })
+
+func withWatchdog() libvmi.Option {
+	return func(vmi *v1.VirtualMachineInstance) {
+		vmi.Spec.Domain.Devices.Watchdog = &v1.Watchdog{
+			Name: "mywatchdog",
```

Comment:
would we want to parametrize the name ?... Does that make *any* sense ?

Reply:
Since it used only once I bet the name means very little or nothing at all. I'm not in the position to decide to drop it from the API schema. If we ever need to create a VM with two watchdog devices we would need to either expose the name as an argument or randomize it. I would not like to worry about it much now.